adoc/bp_postgresql_database_migration.adoc

```diff
@@ -4,33 +4,37 @@
 include::entities.adoc[]
 
 
-{productname} 3 uses postgresql database version 9.4.
-Postgresql version 9.6 has been officially released for {sls} 12 SP3.
-In the near future postgresql 9.6 will become the base version provided by {productname}.
-Currently version 9.4 is hardcoded into SUSE Manager, therefore when installing {productname} it will explicitly use this version.
-This chapter provides guidance on migrating an existing 9.4 database to 9.6 on your {productname} Server.
+{productname} 3 uses PostgreSQL database version 9.4.
+PostgreSQL version 9.6 has been officially released for {sls} 12 SP3, and version 10 for {sls} 12 SP4 and later.
```

Comment:
```suggestion
PostgreSQL version 9.6 has been released for {sls} 12 SP3.
PostgreSQL version 10 has been released for {sls} 12 SP4 and later.
```

Split this just to make it super clear what's going on, otherwise the clauses get a little confusing with all the version numbers.